### DISCUSSION TITLE
can't figure out how to do it?

### DISCUSSION THREAD

**Discussion by aikenadam** [videovillage/community#174] [General]
HI
First I'd like to say, this is the best program I personally could wish for :). Thanks for making it!
Two questions:
1.
Is it possible to change permanently settings for exporting videos? Example: I use Canon RAW videos, and screen show them using decode quality "quarter" as default, to export each video I have to change decode beforehand if I do not want low quality video as a result. How to make exporting with full decode quality every time without hassle of changing it manually?

2. Is it possible to export a bunch of videos from playlist? It would be so good to have that option. let's say to check all the videos I need, make some marks, in and out points and then export them  and go to make a coffee ;)

Again, fantastic program, worth every penny. :)

**Comment by NJ-Digital-Nj** [comment]
is batch process posable
